modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/components/modal/ImportStructureModalContent.tsx

```diff
@@ -0,0 +1,168 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert from '@clayui/alert';
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
+import {ClayInput} from '@clayui/form';
+import ClayModal from '@clayui/modal';
+import {sub} from 'frontend-js-web';
+import React, {useRef, useState} from 'react';
+
+const JSON_EXTENSION = '.json';
+
+export default function ImportStructureModalContent({
+	closeModal,
+}: {
+	closeModal: () => void;
+}) {
+	const [warning, setWarning] = useState(true);
+	const [jsonFile, setJsonFile] = useState<File | null>(null);
+	const fileInputRef = useRef<HTMLInputElement>();
+
+	const resetFileInput = () => {
+		if (fileInputRef && fileInputRef.current) {
+			fileInputRef.current.value = '';
+		}
+
+		setJsonFile(null);
+	};
+
+	const onChangeClick = () => {
+		fileInputRef.current?.click();
+
+		resetFileInput();
+	};
+
+	const handleFileInputChange = async ({
+		target,
+	}: React.ChangeEvent<HTMLInputElement>) => {
+		if (
+			!target.value.endsWith(JSON_EXTENSION) ||
+			!target.files ||
+			target.files?.length === 0
+		) {
+			return;
+		}
+
+		setJsonFile(target.files[0]);
+	};
+
+	return (
+		<>
+			<ClayModal.Header>
+				{Liferay.Language.get('import-and-override-structure')}
+			</ClayModal.Header>
+
+			{warning && (
+				<ClayAlert
+					displayType="warning"
+					onClose={() => {
+						setWarning(false);
+					}}
+					title={`${Liferay.Language.get('warning')}:`}
+					variant="stripe"
+				>
+					{Liferay.Language.get(
+						'import-and-override-structure-warning-message'
+					)}
+				</ClayAlert>
+			)}
+
+			<ClayModal.Body>
+				<ClayInput.Group>
+					<ClayInput.GroupItem>
+						<label htmlFor="jsonInputId">
+							{Liferay.Language.get('json-file')}
+						</label>
+
+						<ClayInput
+							id="jsonInputId"
+							value={jsonFile?.name || ''}
+						/>
+
+						<ClayInput
```

Comment:
We can use a native `<input` for hiding it we don't need the Clay one.

Reply:
done!